controller/backing_image_manager_controller.go

```diff
@@ -1229,15 +1214,15 @@ func (m *BackingImageManagerMonitor) pollAndUpdateBackingImageFileMap() (needSto
 	bim, err := m.ds.GetBackingImageManager(m.Name)
 	if err != nil {
 		if apierrors.IsNotFound(err) {
-			m.log.Info("Stopping monitoring because the backing image manager no longer exists")
+			m.log.Debug("Stopping monitoring because the backing image manager no longer exists")
```

Comment:
Will it only show once? If yes, using `Info` is good to know the image does not exist anymore.

Reply:
We have lots of `apierrors.IsNotFound(err)` 
Do you think we should info it or warn it or debug it? 

From my point of view
Most of the time we try to get the resource because we want to do something
But since it is gone we don't need to do anything, so we can skip it.

If our goal here is to reduce the log size, I think we don't have to log this kind of message.
Moreover, If the resource is gone we should be aware by the missing of the CR in the cluster and no need to check the log.

Reply:
After discussion,
For this case, since it will stop the monitor because of this condition, so we should use `info` here
Thanks @derekbit